### PR TITLE
transport bridge UI

### PR DESCRIPTION
**Pull request by mroz22**
# bridge-node status page 

part of #11532

Features:
- data parity with the old bridge status page
- dark/light theme using autodetect
- support for translations

Todo: 
- design

image

### CODE REVIEW

**Review by komret — APPROVED** [review]

File: packages/transport-bridge/src/ui/components/Device.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                return 'Trezor One (WebUSB)';
+            case 2:
+                return 'Trezor One (WebUSB Bootloader)';
+            case 3:
+                return 'Trezor model T/R';
+            case 4:
+                return 'Trezor model T/R Bootloader';
+            case 5:
+                return 'Emulator';
+            default:
```
Comment (peter-sanderson) [inline packages/transport-bridge/src/ui/components/Device.tsx]: nit: I would prefer to have it strict mapping without having default case

File: packages/transport-bridge/src/http.ts [hunk 2]
```diff
@@ -192,20 +206,88 @@ export class TrezordNode {
...
+                        '.json': 'application/json',
+                        '.css': 'text/css',
+                        '.png': 'image/png',
+                        '.jpg': 'image/jpeg',
+                        '.svg': 'image/svg+xml',
+                    };
+
+                    const { ext } = path.parse(pathname);
+
+                    fs.exists(pathname, function (exist) {
```
Comment (peter-sanderson) [inline packages/transport-bridge/src/http.ts]: nit: arrow function? I think we use (prefer) them everywhere

File: packages/transport-bridge/src/http.ts [hunk 3]
```diff
@@ -192,20 +206,88 @@ export class TrezordNode {
...
+                            res.end(`File ${pathname} not found!`);
+
+                            return;
+                        }
+
+                        // if is a directory search for index file matching the extension
+                        if (fs.statSync(pathname).isDirectory()) pathname += '/index' + ext;
+
+                        // read file from file system
+                        fs.readFile(pathname, function (err, data) {
```
Comment (peter-sanderson) [inline packages/transport-bridge/src/http.ts]: nit: arrow function? I think we use (prefer) them everywhere

**Review by peter-sanderson — APPROVED** [review]
just nits

**Comment by mroz22** [comment]
forced pushed the nitpicks

**Pull request merged**